src/components/picker/picker.tsx

```diff
@@ -322,7 +322,7 @@ export class Picker {
         const name = getIconName(listItem.icon);
         const color = getIconFillColor(listItem.icon, listItem.iconColor);
         const value = listItem.value;
-        const id = typeof value !== 'object' ? value : value.id;
+        const id = (!!value && typeof value === 'object') ? value.id : value;
```

Comment:
I was wondering if the root of the problem is that `ListItem` interface doesn't have `id`, 

https://github.com/Lundalogik/lime-elements/blob/4b731c74df02b5e2519bb81f803a90266d7f9948/src/components/list/list-item.types.ts#L8

while the `Chip` interface which is used in `limle-chip-set` expects `id`s:
https://github.com/Lundalogik/lime-elements/blob/4b731c74df02b5e2519bb81f803a90266d7f9948/src/components/chip-set/chip.types.ts#L10

but in this example that you were testing on, the consumer has to include the `id` in the object, due to the lack of `id` in the `ListItem` interface that is used in `limel-picker` (picker is using chip-set).

Hope you followed what I was trying to say. But I just wonder if we should revisit `ListItem` instead.

Reply:
I'm changing a bit, I will add a fixup soon, so you can check if it is still relevant

Reply:
I haven't really looked into the problem or the solution, I just saw this was some missmatch between what LiteItem interface allows the users to do, and what Chip interface has, which forces the users to implement the code in a way that becomes broken. But what is a good fix, I have no idea. I can't say where this problem should be fixed.